cpp/ps0000/solution 1.cpp

```diff
@@ -1,7 +1,9 @@
-#include<iostream>
+#include <iostream> // 2/2
 using namespace std;
-int main ()
-[
-cout <<"Hello,World!/n";
-return 0;
-]
\ No newline at end of file
+
+int main () // 0.5/1
+{
+  cout << "Hello, World!/n"; // 3.5/4
```

Comment:
- [x] used `<<` for outputting instead of `>>`
- [x] used `\n` or `std::endl` in the right way
- [x] avoided syntax error by typing `;` at the end of the line
- [ ] output is as required `Hello, World!` in a single line